### PR TITLE
journals: fix lowercase_journal_titles type

### PR DESCRIPTION
**Pull request by drjova**
Signed-off-by: Harris Tzovanakis <me@drjova.com>

### CODE REVIEW

File: backend/inspirehep/search/mappings/v7/records/journals.json [hunk 1]
```diff
@@ -245,10 +245,25 @@
       "normalizer": {
         "lowercase_normalizer": {
```
Comment (michamos) [inline backend/inspirehep/search/mappings/v7/records/journals.json:246]: this affects other fields too that are using it. Maybe create a new specific normalizer for this instead?

Comment (drjova) [reply backend/inspirehep/search/mappings/v7/records/journals.json:246]: I was thinking about it, it's only affecting `_collections` which will not be affected but I agree it's cleaner to separate it.

**Comment by drjova** [comment]
@michamos done

**Comment by drjova** [comment]
@michamos I have added a test that covers explicity for white spaces as well

**Pull request merged**